CppTicTacToe/tictactoe.cpp

```diff
@@ -13,6 +13,7 @@ using namespace std;
 
 // main global variables and initialization follow here
 
+// Test
 int round_counter = 1;
```

Comment:
The first thing I want to say about this code is good advice for C++ and programming in general. Any time there are global variables ( usually just called globals) this is a very bad sign.

I could give you articles on why this is bad, or rant about it, but I'll try to be brief. Global's live for the lifetime of your program, so they are inherently inefficient. More importantly, Globals can be used anywhere. If you modify game_over in lots of places in your program it becomes hard to track down whether it was set to true at the right time, or if you need to check right now if its still false because it might have been set to true somewhere else.

Reply:
Eliminating your globals is a problem that will take work. Thankfully, it is rooted in dealing with the other aspects of your code that we can work on. You will see how we can eliminate globals as we reform your functions into things with responsibilities :)

Reply:
This makes sense about globals.

I guess I figured that in a smaller program like this, it didn't matter to just make most of them global scope. But it also makes sense to get closer to best practices even with small projects.

With a program like this, it feels like there is a pretty linear and logical way in which the globals are changed. It will be good practice to think about how not using globals can make the code more semantically focused, though.